src/nvim/main.c

```diff
@@ -259,6 +261,35 @@ int main(int argc, char **argv)
   command_line_scan(&params);
   server_init(params.listen_addr);
 
+  if (params.cs_remote) {
+    const char *env_addr = os_getenv("NVIM_LISTEN_ADDRESS");
+    CallbackReader on_data = CALLBACK_READER_INIT;
+    const char *error = NULL;
+    uint64_t rc_id = channel_connect(false, env_addr, true,
+                                     on_data, 50, &error);
+    if (!rc_id) {
+      exit(0);
+    }
+
+    int t_argc = params.cs_remote;
+    Array args = ARRAY_DICT_INIT;
+    String arg_s = cstr_to_string(argv[t_argc]);
+    do {
+      arg_s = cstr_to_string(argv[t_argc]);
+      ADD(args, STRING_OBJ(arg_s));
+    }while(++t_argc < argc);
+
+
+    Error err;
+    Array a = ARRAY_DICT_INIT;
+    ADD(a, INTEGER_OBJ((int)rc_id));
+    ADD(a, ARRAY_OBJ(args));
+    String s = cstr_to_string("return vim._cs_remote(select(1, ...))");
```

Comment:
`select(1, ...)` is equivalent to `...`. Please change it in api/vim.c also, otherwise it will get cargo-culted without reason....